lib/usd/utils/DiffCore.h

```diff
@@ -13,15 +13,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-#pragma once
+#ifndef UTILS_DIFFCORE_H
+#define UTILS_DIFFCORE_H
 
-#include "AL/usd/utils/Api.h"
-#include "AL/usd/utils/ALHalf.h"
+#include "Api.h"
+#include "ALHalf.h"
```

Comment:
again - can we not change this until coding standards have been agreed upon?

Reply:
To be sure, your ask is to change it to:
```
#include "UsdUtils\Api.h"
#include "UsdUtils\ALHalf.h"
```